docs/config-olap-databases.md

```diff
@@ -0,0 +1,30 @@
+---
+id: config-olap-database
+title: OLAP Database Configuration
+sidebar_label: OLAP Database
+---
+
+## Section
+
+`olap_database:` The OLAP (online analytical processing) database section configures the OLAP database that BuildBuddy uses to enable the Trends page. **Optional**
+
+Note: in order to use OLAP database for the Trends page, `app.enable_read_from_olap_db` and
+`app.enable_write_to_olapdb` needs to be set to `true`
+
+## Options
+
+**Optional**
+
+- `data_source` This is a connection string used by the database driver to connect to the database. ClickHouse database is supported.
```

Comment:
nit: is there some documentation that you can link to for what this connection string looks like?

Reply:
I included an example below.